### ISSUE TITLE
Osu!lazer mobile touchsreen freezes w/ game crashing

### ISSUE THREAD

**Issue by Pneu666** [ppy/osu#12215]
When i play, for 3 minutes or more, the game freezer for a sec and then the touch stop working, making the game crash

https://user-images.githubusercontent.com/81386229/112854141-ec081400-9083-11eb-9e37-7870561aaaef.mp4

**osu!lazer version:** 2021.323.0 

**Logs:** 
[database.log](https://github.com/ppy/osu/files/6222527/database.log)
[network.log](https://github.com/ppy/osu/files/6222528/network.log)
[performance.log](https://github.com/ppy/osu/files/6222529/performance.log)
[runtime.log](https://github.com/ppy/osu/files/6222530/runtime.log)

**Device Information:** 
Xiaomi Redmi Note 6 Pro
4 GB RAM / 64 ROM
Snapdragon 636
Adreno 509
Android 9.1 / Miui 12.0.1

**Comment by frenzibyte** [comment]
The crash looks to be a duplicate of https://github.com/ppy/osu-framework/issues/2669, do you always get this? if so, what steps did you get through to do it? it would be helpful in potentially solving this since this type of crash has been reported once 2 years ago.

**Comment by Pneu666** [comment]
> The crash looks to be a duplicate of [ppy/osu-framework#2669](https://github.com/ppy/osu-framework/issues/2669), do you always get this? if so, what steps did you get through to do it? it would be helpful to potentially solve this since this type of crash, as it was reported only once over a year ago.

Question 1: so, almost every time i get this
Question 2: i just play casually and its random this crashing, probably when the game start to lag a bit it has a chance of crashing this way

**Comment by frenzibyte** [comment]
Hmm, well the crash is related to android game view receiving update frames while the elapsed time from last update is 0ms, so theoretically having the game lag may be related.

Thanks for answering, closing this as a duplicate of the aforementioned issue.
